tox.ini

```diff
@@ -38,15 +36,3 @@ commands =
 [testenv:readme-py27]
 commands = python setup.py check -r -s
 deps = readme_renderer
-
-[testenv:flake8-py27]
-commands = flake8 configurations tests
-deps = flake8
-
-[testenv:flake8-py36]
-commands = flake8 configurations tests
-deps = flake8
-
-[flake8]
-exclude = .tox
-ignore = E501,E127,E128,E124,W503
```

Comment:
I think it is still useful to have config for this in tox.ini.
Could become `check` / `checkqa` and also run the other tools then.

Reply:
You don't need the flake8 config in the tox file for that, but you do need a tox environment to run flake8. I will add it again, but omit it from Travis-CI.

Reply:
You can run the checks locally now, via `tox -e checkqa`